### PR TITLE
Port over com.ibm.ws.logging.hpel_fat tests

### PR DESCRIPTION
**Pull request by Yushan-Lin**
Referring to https://github.ibm.com/was-liberty/WS-CD-Open/issues/20452

### CODE REVIEW

**Comment by Yushan-Lin** [comment]
#build

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_KztqMHjWEeqVMooXUPL4-g

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_Qmh4MHjoEeqVMooXUPL4-g

File: dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/logging/hpel/fat/HPELDataDirFalsePositiveWarningTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
+//%Z% %I% %W% %G% %U% [%H% %T%]
```
Comment (fmhwong) [inline dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/logging/hpel/fat/HPELDataDirFalsePositiveWarningTest.java:1]: Missing copyright statement

File: dev/com.ibm.ws.logging.hpel_fat/test-applications/HpelFat/src/com/ibm/ws/logging/hpel/servlet/MyCustomHandler.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.ibm.ws.logging.hpel.servlet;
```
Comment (fmhwong) [inline dev/com.ibm.ws.logging.hpel_fat/test-applications/HpelFat/src/com/ibm/ws/logging/hpel/servlet/MyCustomHandler.java:1]: Missing copyright statement

**Comment by fmhwong** [comment]
I think most of the files were ported from WS-CD.  Can you describe what you have changed in the porting?

**Comment by Yushan-Lin** [comment]
@fmhwong main thing that was changed was I removed `dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/fat/hpel/setup/HpelSetup.java` class and updated the tests so nothing is dependent on it anymore.

**Comment by Yushan-Lin** [comment]
#run-libby-bot

**Comment by Yushan-Lin** [comment]
#run-libby-bot

**Comment by Yushan-Lin** [comment]
#run-libby-bot

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_S6fj0IDIEeqspO8IkKlEbg

**Pull request merged**